src/Microsoft.IO.Redist/src/Microsoft.IO.Redist.csproj

```diff
@@ -0,0 +1,237 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <AssemblyName>Microsoft.IO.Redist</AssemblyName>
+    <IsPartialFacadeAssembly>false</IsPartialFacadeAssembly>
+    <ProjectGuid>{FEF5811F-ED50-4407-A6B9-885EBD3206FB}</ProjectGuid>
+    <Configurations>netfx-Debug;netfx-Release</Configurations>
+    <DefineConstants>$(DefineConstants);MS_IO_REDIST</DefineConstants>
+    <LangVersion>7.3</LangVersion>
+    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
+    <ClsCompliant>false</ClsCompliant>
+    <UWPCompatible Condition="'$(TargetGroup)' == 'uap' or '$(TargetGroup)' == 'uapaot'">true</UWPCompatible>
```

Comment:
nit: can be removed

Reply:
I kept it to avoid the UWP warnings.

Reply:
I don't expect you'll get those now that you only target netfx.